Add support for outputting jsdocs

adrianord opened the pull request:
Adds an option for including jsdoc comments for use with LSPs in editors.

I initially tried to keep the jsdocs inline with everything else, but unfortunately they need to be expanded into multiple lines for compatibility with different editors. I tested in `neovim` using `ts-ls` and had to expand in order for the comments to show in hover tool tips.

Fixes #114

StefanTerdell commented:
Oh, cool! Looks like it works pretty well!

Required: Add some documentation to the readme and I think we're good to merge.

Optional: Regarding inlining the comments, maybe onelining would be a nice compromise (if the description contains no newlines of course)?

```
/** Description for nestedProp */
nestedProp: z.blah
```

Works in helix with tls

adrianord commented:
Thanks for the review 😄 

I've added some documentation for the `withJsdocs` flag to the README, let me know if you'd like me to document a bit more about the expected output of the jsdocs.

Onelining the single line descriptions sounded like a good idea so I've gone ahead and implemented that, too. Also double checked that that still works for neovim.
